### ISSUE TITLE
Port binding.host_id set to null? 

### ISSUE THREAD

**Issue by GregoryVds** [terraform-provider-openstack/terraform-provider-openstack#1082]
Hello,

If I run `terraform plan` immediately after the deployment of my infrastructure, without any change, I see that the port `binding.host_id` would be set to `null`. I would expect this sub-subsequent `plan` to be give me a no-op.

I only want to specify the `vnic_type` for the port, but I don't want to specify the `host_id` and I left this empty in my Terraform resource definition. 

Why is it resetting the `host_id`? 
(If I apply this plan, the `binding.host_id` is set to null, my port enters `DOWN` status and I loose connectivity).

Thanks in advance,
Gregory

### Terraform Version

Terraform v0.13.4
Terraform Orovider Openstack v1.32.0

### Affected Resource(s)
- openstack_networking_port_v2

``` 
resource "openstack_networking_port_v2" "node_main" {
  name           = "node-main"
  admin_state_up = "true"
  port_security_enabled = "false"
  network_id = var.network.main_id 

  binding {
    vnic_type = var.perf ? "direct" : "normal"
  }
}

resource "openstack_networking_port_v2" "node_int" {
  name           = "node-int"
  admin_state_up = "true"
  port_security_enabled = "false"
  network_id = var.network.int_id

  binding {
    vnic_type = var.perf ? "direct" : "normal"
  }
}

resource "openstack_networking_port_v2" "node_slave" {
  name           = "node-slave"
  admin_state_up = "true"
  port_security_enabled = "false"
  network_id = var.network.slave_id

  binding {
    vnic_type = var.perf ? "direct" : "normal"
  }
}
``` 

### Debug Output

https://gist.github.com/GregoryVds/5e02af04cea43598235e5670dc504eb0

### Expected Behavior

I expect this `plan` to be a no-op and the ports to be untouched. My infrastructure has just been freshly deployed and I didn't make any config change.

### Actual Behavior

It seems the port `binding.host_id` is set to `NULL`. 
Port then enters status `DOWN` and connectivity is lost.

**Comment by GregoryVds** [comment]
I did some more testing, and the problem does arise if I remember the whole `binding` section in the Terraform resource definition.

**Comment by ozerovandrei** [comment]
Fixed in #1084
